mobile/src/reducers/people/people.reducer.ts

```diff
@@ -85,6 +85,21 @@ export const peopleReducer: Reducer<PeopleState> = (state = initState, action: P
                     const depsAndMeta = departmentsBranchFromDepartmentWithId(departmentId, state.departments);
                     return {...state, currentFocusedDepartmentId: departmentId, departmentsBranch: depsAndMeta.departmentsLineup, departmentsLists: depsAndMeta.departmentsLists};
                 }
+            } else if (action.routeName === 'CurrentProfile') {
```

Comment:
Revert this file?

Reply:
ok, I thought I've cleaned this one, seems to be did not